lib/fizzbuzz.rb

```diff
@@ -2,10 +2,16 @@ class SuperFizzBuzz
 
   def run(input)
 
-    #Implement your code here
-
-  end
-
+   if input % 3 == 0 && input % 5 == 0
+       return 'FizzBuzz'
```

Comment:
Explicit `return`s are typically avoided by Rubyists. This if/elsif block could just have values inside each branch.

This top one would look like:

```ruby
if input % 3 == 0 && input % 5 == 0
  'FizzBuzz'
```

The value in the branch that is evaluated becomes the return value of the whole if/elsif, which in turn becomes the implicit return value for the method.